portfolio/src/main/webapp/script.js

```diff
@@ -12,12 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-
 // Carousel
-
 $('.carousel').carousel({
   interval: 2000
 });
+/* a map display */
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 40.693320, lng:-73.988424}, 
+      zoom: 25, 
+      mapTypeId: "satellite"
+      });
+}
+/* location */
```

Comment:
This comment goes with the `var Roccos...` right? That would be clearer if you had the blank line above the comment and no blank line between the comment and `var Roccos...`